easybuild/easyconfigs/b/Blender/Blender-3.0.0-binary.eb

```diff
@@ -0,0 +1,36 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'PackedBinary'
+
+name = 'Blender'
+version = '3.0.0'
+versionsuffix = '-binary'
```

Comment:
Maybe `x64` is a better versionsuffix?

Reply:
or even better: `-linux-x64` ?

Reply:
@robert-mijakovic if you don't mind I went ahead and did the change for you.

Reply:
@smoors Oh, I don't. I haven't even noticed the change before. Thank you.